src/components/sidebars/Features.js

```diff
@@ -0,0 +1,18 @@
+import React, {Component} from "react";
+import FeaturesList from "../../containers/sidebars/FeaturesList";
+
+class Features extends Component {
```

Comment:
I would consider using pure functions here instead of component

Reply:
Done. I've created a file in `functions` directory that returns the list.